types/jasmine/index.d.ts

```diff
@@ -1017,14 +1036,20 @@ declare namespace jasmine {
         jasmineDone?(runDetails: JasmineDoneInfo, done?: () => void): void | Promise<void>;
     }
 
-    type SpecFunction = (spec?: Spec) => void;
```

Comment:
Add a deprecated tag just like above.

Reply:
done, good shout, it could be used by someone, even if not used in DT
Amended